core/src/main/scala/kafka/server/ClientQuotaManager.scala

```diff
@@ -129,7 +129,7 @@ case class QuotaEntity(quotaId: QuotaId, sanitizedUser: String, clientId: String
  * @param apiKey API Key for the request
  * @param time @Time object to use
  */
-class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
+final class ClientQuotaManager(private val config: ClientQuotaManagerConfig,
```

Comment:
findbugs warns about starting threads in a class's constructor, *unless* that class is final.  The reason for this is that if it is ever subclassed, the thread would be started in the superclass's constructor, before any of the derived fields had been initialized, which could lead to surprising results.  It seemed easier to just make the class final than to add a suppression or refactor the class to not do that.